Tests/MessagingPushAPN/APITest.swift

```diff
@@ -27,13 +27,11 @@ class MessagingPushAPNAPITest: UnitTest {
 
         MessagingPush.shared.application(
             "",
-            didFailToRegisterForRemoteNotificationsWithError: CustomerIOError
-                .notInitialized
+            didFailToRegisterForRemoteNotificationsWithError: MessagingPushError.registrationFailed
```

Comment:
Replaced with generic error as `CustomerIOError` is no longer accessible by `MessagingPush`